sky/resources.py

```diff
@@ -346,20 +361,6 @@ def _try_validate_accelerators(self) -> None:
             # because e.g., the instance may have 4 GPUs, while the task
             # specifies to use 1 GPU.
 
-        # Validate whether accelerator is available in specified region/zone.
-        acc, acc_count = list(acc_requested.items())[0]
-        if self.region is not None or self.zone is not None:
-            if not self._cloud.accelerator_in_region_or_zone(
-                    acc, acc_count, self.region, self.zone):
-                error_str = (f'Accelerator "{acc}" is not available in '
-                             '"{}" region/zone.')
-                if self.zone:
-                    error_str = error_str.format(self.zone)
-                else:
-                    error_str = error_str.format(self.region)
-                with ux_utils.print_exception_no_traceback():
-                    raise ValueError(error_str)
```

Comment:
Good point. While the optimizer can also filter out the invalid combinations of accelerators and locations, let's keep it so that we can catch and print the error early.